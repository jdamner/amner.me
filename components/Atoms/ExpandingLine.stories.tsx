import type { Meta, StoryObj } from "@storybook/nextjs";

import Expandingline from "./ExpandingLine";

type Story = StoryObj<typeof Expandingline>;

export default {
  title: "Atoms/Expandingline",
  component: Expandingline,
} as Meta;

export const Default: Story = { args: { reverse: false } };
export const Reverse: Story = { args: { reverse: true } };
