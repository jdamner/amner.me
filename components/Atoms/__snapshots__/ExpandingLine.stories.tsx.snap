// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atoms/Expandingline Default smoke-test 1`] = `
<div class="bg-slate-100 dark:bg-slate-900 text-slate-900 dark:text-slate-300">
  <div class="bg-slate-800 dark:bg-slate-300 h-1 w-full"
       style="max-width: 100%;"
  >
  </div>
</div>
`;

exports[`Atoms/Expandingline Reverse smoke-test 1`] = `
<div class="bg-slate-100 dark:bg-slate-900 text-slate-900 dark:text-slate-300">
  <div class="bg-slate-800 dark:bg-slate-300 h-1 w-full"
       style="margin-left: auto; max-width: 100%;"
  >
  </div>
</div>
`;
